resources/dicts/patrols/general/medcat.json

```diff
@@ -279,5 +279,644 @@
+    "chance_of_success": 30,
+    "exp": 30,
+    "success_text": [
+        "The cats take turns veering off for herbs, allowing the others time to chat with each other.  They manage to avoid any rain, and by the time they loop back to camp, each cat has a small bundle of herbs in their jaws, and a small bundle of contentment in their hearts.",
+        "The patrol decides to collect herbs first, so they have time to idle and chat before heading back to camp. They sit themselves in the cover of a tree to share tongues and chat out of the rain, and by the time they're ready to go home, every cat is feeling closer to one another.",
+        "As the cats walk and chat, s_c begins to lag behind. p_l looks back to ask them what's up, and spots them veering down a side path. p_l motions for the others as they follow s_c, and find them purring loudly at the sight of a huge herb patch! There's far more than the patrol can carry, and they know this will keep the Clan healthy for ages."
+    ],
+    "fail_text": [
+        "It starts pouring rain while they're out, and even p_l has to admit defeat and lead them home."
+    ],
+    "win_skills": [
+        "keen eye",
+        "very smart",
+        "extremely smart"
+    ],
+    "min_cats": 3,
+    "max_cats": 6,
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_appgroup_randomherb1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": [
+        "med_cat",
+        "med_only",
+        "apprentice",
+        "three_apprentices",
+        "random_herbs",
+        "many_herbs1",
+        "comfort",
+        "trust",
+        "platonic",
+        "no_change_fail"
+    ],
+    "intro_text": "The trio of apprentices strolls through c_n's territory, chatting amicably amongst themselves. The senior medicine cats are all busy back at camp, so the young cats get the chance to be out on their own today, which has everyone excited!",
+    "decline_text": "Before they get very far, one of their Clanmates catches up to inform them that they're wanted back at camp. Oh well, maybe they'll get the chance again another day.",
+    "chance_of_success": 25,
+    "exp": 30,
+    "success_text": [
+        "app1 dramatically flops over into a sunbeam, proclaiming that they need to get some sunshine before hunting for herbs. app2 purrs and flops beside them, while app3 rolls their eyes and loafs nearby. Once they've all warmed up, they renew their search with vigor, and return home content and with herbs.",
+        "app1 is the first to find some herbs, and smugly waves them in the faces of app2 and app3 - after that it's a bit of a competition, with each cat trying to out-compete one other. It's great for the Clan - in the end, they come back to camp with a sizeable haul!"
+    ],
+    "fail_text": [
+        "The apprentices are too caught up in chatting with (and riling up) one another - they don't collect a single leaf."
```

Comment:
Maybe use hyphens instead of parentheses?
"chatting with - and riling up - one another"